puppeteer-screenshot/public/index.html

```diff
@@ -0,0 +1,18 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Screenshot as a service</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <h1>Screenshot as a service</h1>
+        <p>Please provide a path to a website or choose one of the following:</p>
+        <ul>
+            <li><a href="/google.com">/google.com</a></li>
+            <li><a href="/zeit.co/blog?type=png">/zeit.co/blog?type=png</a></li>
+            <li><a href="/zeit.co/about?fullPage=true">/zeit.co/about?fullPage=true</a></li>
+            <li><a href="/ceriously.com?type=jpeg&quality=75&fullPage=true">/ceriously.com?type=jpeg&quality=75&fullPage=true</a></li>
```

Comment:
```suggestion
             <li><a href="/example.com?type=jpeg&quality=75&fullPage=true">/example.com?type=jpeg&quality=75&fullPage=true</a></li>
```

I would change this one to example.com instead of your personal website 🕵️